include/pinocchio/spatial/inertia.hpp

```diff
@@ -228,9 +229,16 @@ namespace pinocchio
       X.disp(os);
       return os;
     }
+  };
 
-  }; // class InertiaBase
+  // Forward declare the PseudoInertia and LogCholeskyParameters structs
+  template<typename Scalar, int Options>
+  struct PseudoInertiaTpl;
 
+  template<typename Scalar, int Options>
+  struct LogCholeskyParametersTpl;
```

Comment:
You can move this forward declaration in `pinocchio/spatial/fwd.hpp:60` and default initialize Options with `context::Options`.